src/pybind/mgr/dashboard/frontend/src/app/shared/services/summary.service.ts

```diff
@@ -32,7 +32,11 @@ export class SummaryService {
   }
 
   private retrieveSummaryObservable(): Observable<Summary> {
-    return this.http.get<Summary>('api/summary');
+    return this.http.get<Summary>('api/summary').pipe(
+      catchError((_) => {
+        return of(null);
```

Comment:
Why this ? Are we getting any errors ?

Reply:
The main issue here was that the summary service stopped polling after the block-ui  got completed so I have removed this code from here, instead I am starting the summary endpoint polling again after the block UI completion, so it solves the issue broadly whenever block-ui is called.